src/System.Text.Json/tests/TestClasses.ClassWithComplexObjects.cs

```diff
@@ -54,6 +54,8 @@ public void Verify()
             JsonProperty property = jsonObject.EnumerateObject().First();
             Assert.Equal("NestedArray", property.Name);
             ValidateArray(property.Value);
+            Assert.Equal(default, jsonObject.ValueEquals("NestedArray"));
+            Assert.Equal(default, property.NameEquals("NestedArray"));
```

Comment:
A) `false` is better than `default`
B) just use Assert.False (which allows for a message to figure out which test it was)